gramps/plugins/lib/maps/geography.py

```diff
@@ -397,8 +394,7 @@ def build_nav_menu(self, osm, event, lat, lon):
         add_item = Gtk.MenuItem(label=title)
         menu.append(add_item)
 
-        self.changemap = Gtk.Menu()
```

Comment:
We had an issue in the past, where a menus failed to display if they were stored in local variables.  I'm not sure if this has been fixed on all supported Gtk versions yet.

Reply:
When I tested this PR, I didn't see problems. I suppose this has been solved.